config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml

```diff
@@ -97,7 +97,7 @@ presets:
   - name: KUBELET_TEST_ARGS
     value: "--enable-debugging-handlers"
   - name: KUBEPROXY_TEST_ARGS
-    value: "--profiling"
+    value: "--profiling --metrics-bind-address=0.0.0.0"
```

Comment:
This looks fine to me for now. But can you please open an issue for figure out what's our longer-term solution?

Reply:
Done - https://github.com/kubernetes/kubernetes/issues/74011

Added change the override depending on the issue outcome.